web-calculator.css

```diff
@@ -0,0 +1,71 @@
+body{
+    background-image: url(https://i.pinimg.com/originals/1e/ef/c5/1eefc5567dc5c55f67ef6849b9235582.jpg), url();
+    background-size: cover;
+    margin:350;
+    position: relative;
+}
+h1{
+    font-family: tahoma, sans-serif;
+    font-size: medium;
+    color:#00FFFF;
+    text-align: center;
+} 
+#a{
+    position: relative;
+    display:block;
```

Comment:
You could make these divs and they be block elements instead of making them spans but then overriding their natural display state. Also, `<label>` was actually probably what you were looking for.